lib/browser/api/browser-actions.js

```diff
@@ -62,6 +64,8 @@ module.exports = {
   },
 
   getPopup: (extensionId, details) => {
-    return getBrowserAction(extensionId, details).popup
+    const action1 = getBrowserAction(extensionId, details)
+    const action2 = getBrowserAction(extensionId, {})
```

Comment:
We might consider moving this call to `getBrowserAction` behind the `||` on the next line. That way, if `action1` is all that was needed, we don't make an unnecessary second call.